src/gamma/main.d

```diff
@@ -250,3 +254,11 @@ void build(string[] fileNames, string outputDirectory)
     if (status)
         exit(status);
 }
+
+string executableName(const string name)
+{
+    version(Windows)
+        return name ~ ".exe";
+    else
+        return name;
+}
```

Comment:
Get rid of this red "minus in circle": in VS Code, enable "insertFinalNewline"

Reply:
I do not see the red circle well in my light theme. 😉

Reply:
Thank for the configuration hint regarding "insertFinalNewline"! Didn't know it.

Reply:
Done with 49328b890081262840a0dd7cace4f266e74dbee1.